package.json

```diff
@@ -28,39 +31,44 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
-    "ember-browserify": "1.1.11",
-    "ember-cli": "~2.6.0",
-    "ember-cli-app-version": "1.0.0",
-    "ember-cli-blanket": "0.9.5",
+    "ember-browserify": "~1.1.0",
+    "ember-cli": "~2.8.0",
+    "ember-cli-app-version": "^1.0.0",
+    "ember-cli-code-coverage": "~0.3.0",
     "ember-cli-dependency-checker": "^1.2.0",
-    "ember-cli-deploy": "1.0.0-beta.1",
+    "ember-cli-deploy": "~0.6.4",
```

Comment:
I believe ember-cli-deploy + ember-cli-deploy-gh-pages aren't necessary for the github repos